htdocs/install/include/functions.php

```
@@ -256,6 +256,54 @@ function xoPhpVersion()
+ * @param XoopsInstallWizard $wizard
+ * @param string             $ext     extension name (e.g. 'mysqli')
+ * @param string             $label   human-readable label (e.g. 'MySQLi')
+ * @param string[]           $symbols functions/classes that must also exist;
+ *                                    catches partial builds where the
+ *                                    extension reports loaded but a symbol
+ *                                    the caller uses (e.g. mysqli_report) is
+ *                                    not actually available
+ *
+ * @return void
+ */
+function xoInstallerRequireExtension($wizard, $ext, $label, array $symbols = [])
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$wizard'. (undefined)

(UnusedFormalParameter)